include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -424,17 +424,22 @@ def TTNN_MaxOp : TTNN_ReductionOp<"max"> {
   }];
 }
 
-def TTNN_EmbeddingOp : TTNN_Op<"embedding"> {
+def TTNN_EmbeddingOp : TTNN_NamedDPSOp<"embedding"> {
```

Comment:
I believe @jnie-TT covers this in this PR:
https://github.com/tenstorrent/tt-mlir/pull/1128/files

Reply:
Yes, I copied over his changes to this PR since gather requires embedding to work.

Reply:
FYI this has been merged to main